modules/ROOT/pages/websocket/websocket-connector-cloudhub.adoc

```diff
@@ -0,0 +1,42 @@
+= Using Websockets Connector in CloudHub
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can deploy Mule apps that contain inbound WebSocket endpoints, outbound WebSocket endpoints, or both to CloudHub.
+
+If your app has inbound WebSocket endpoints only, to deploy the app in CloudHub you must:
+
+. Configure an Anypoint Virtual Private Cloud (VPC).
+. Set up a dedicated load balancer to route WS and WSS traffic through the Mule app.
+
+To set up the inbound WebSocket endpoints:
+
+. Configure the WebSockets Inbound Listener (`<ws:inbound-listener>`) to listen on one of the following ports:
+* `0.0.0.0`.
+///
+when is this port used?
+///
+* To communicate over WS, set the listener port to `8091`.
+* To communicate over WSS, set the listener port to `8092`.
+. Deploy your app to the VPC.
+. Set up the dedicated load balancer's URL mapping rules to associate a URI to a specific app.
++
+For example, if your app is named 'my-websocket-mule-app', you can define a rule like the following:
++
+* Input Path: `/my-websocket-mule-app/`
+* Target App: `my-websocket-mule-app`
+* Output Path: `/`
+* Protocol: `ws`
+
+This rule routes all requests made to your dedicated load balancer's URL and the specified input path to your Mule app's root path `/`. The load balancer's name determines its assigned URL, so for this example, if you name your load balancer 'mylb', then you would need to browse to: `+ws://mylb.lb.anypointdns.net:80/my-websocket-mule-app/+` to connect to your WebSocket Listener.
+
+The dedicated load balancer forwards all requests issued to port 80 to your app's port 8091, which is destined for all WS connections. When using WSS, however, issue your requests to port 443, and these requests are forwarded to your Mule app's port 8092.
+
+
+=== See Also
+
+* https://help.mulesoft.com[MuleSoft Help Center]
+* https://docs.mulesoft.com/runtime-manager/vpc-architecture-concept
```

Comment:
These should be xrefs but I won't quibble over this as long as they work. We can iterate.